classes/ror/Ror.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * @file classes/ror/Ror.php
+ *
+ * Copyright (c) 2024 Simon Fraser University
+ * Copyright (c) 2024 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class Ror
+ *
+ * @ingroup ror
+ *
+ * @see DAO
+ *
+ * @brief Basic class describing a ror.
+ */
+
+namespace PKP\ror;
+
+use PKP\core\DataObject;
+
+class Ror extends DataObject
+{
+    public const STATUS_ACTIVE = 1;
+    public const STATUS_INACTIVE = 0;
```

Comment:
I believe we could add the getters and setters -- I think we decided to use them so that we better support type hinting...